YUP253/About.swift

```diff
@@ -33,6 +33,7 @@ struct About: View {
                        
```

Comment:
Padding of 200 overflows screen on older devices, 
* Recommend changing the Image padding to a smaller value (the desired top padding)
* Then to automatically line the rest of the items at the bottom, use the `Spacer(minLength: 0)` in line 32